dev/io.openliberty.checkpoint_fat_jms/fat/src/io/openliberty/checkpoint/messaging/JMS20/fat/FATSuite.java

```diff
@@ -0,0 +1,92 @@
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.checkpoint.messaging.JMS20.fat;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.PrintWriter;
+import java.io.UncheckedIOException;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.function.Consumer;
+
+import org.junit.ClassRule;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+import componenttest.annotation.SkipIfCheckpointNotSupported;
+import componenttest.rules.repeater.JakartaEE9Action;
+import componenttest.rules.repeater.RepeatTests;
+import componenttest.topology.impl.LibertyServer;
+import io.openliberty.checkpoint.messaging.JMS20.fat.ContextInject.JMSContextInjectTest;
+import io.openliberty.checkpoint.messaging.JMS20.fat.DurableUnshared.DurableUnsharedTest;
+import io.openliberty.checkpoint.messaging.JMS20.fat.JMSConsumerTest.JMSConsumerTest_118076;
+import io.openliberty.checkpoint.messaging.JMS20.fat.JMSConsumerTest.JMSConsumerTest_118077;
+import io.openliberty.checkpoint.messaging.JMS20.fat.JMSContextTest.JMSEjbJarXmlMdbTest;
+import io.openliberty.checkpoint.messaging.JMS20.fat.JMSContextTest.JMSRedeliveryTest_120846;
+import io.openliberty.checkpoint.messaging.JMS20.fat.JMSDCFTest.JMSDCFTest;
+import io.openliberty.checkpoint.messaging.JMS20.fat.JMSDCFTest.JMSDCFVarTest;
+import io.openliberty.checkpoint.messaging.JMS20.fat.JMSMBeanTest.JMSMBeanTest;
+import io.openliberty.checkpoint.messaging.JMS20.fat.JMSProducerTest.JMSProducerTest_118071;
+import io.openliberty.checkpoint.messaging.JMS20.fat.JMSProducerTest.JMSProducerTest_118073;
+import io.openliberty.checkpoint.messaging.JMS20.fat.JMSProducerTest.JMSProducer_Test118073;
+import io.openliberty.checkpoint.messaging.JMS20.fat.SharedSubscription.SharedSubscriptionTest_129623;
+import io.openliberty.checkpoint.messaging.JMS20.fat.SharedSubscription.SharedSubscriptionTest_129626;
+import io.openliberty.checkpoint.messaging.JMS20.fat.SharedSubscription.SharedSubscriptionWithMsgSelTest_129623;
+import io.openliberty.checkpoint.messaging.JMS20.fat.SharedSubscription.SharedSubscriptionWithMsgSelTest_129626;
+
+@SkipIfCheckpointNotSupported
+@RunWith(Suite.class)
+@SuiteClasses({
+                DummyTest.class,
+
+                  BucketSet1CpClientTest.class,
+                  BucketSet2CpClientTest.class,
+                  BucketSet1CpEngineTest.class,
+                  BucketSet2CpEngineTest.class,
+
+})
+
+public class FATSuite {
+
```

Comment:
As you go, clean up this class.  Editor shows lots of unnecessary whitespace, mixed use of tabs and spaces.  Methods look good.